_data-prepper/pipelines/configuration/processors/parse-json-processor.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+title: Parse_JSON
```

Comment:
It looks like we use "Parse_JSON" as the title, "Parse JSON" as H1, and `Parse_json` as the name of the processor. Should all of these be consistent?

Reply:
They definitely should. Thanks for catching this.